packages/fxa-content-server/app/scripts/lib/glean/account.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v7.2.1. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
@badboy All the Glean generated files have this comment.  But I don't see the advantage of having to re-generate these files every time we build this package or on git checkout.  What's the harm in committing them?

Reply:
Unrelated but maybe slightly&hellip; if these are all auto-generated, do we want to make sure they get ignored by Prettier/ESLint to avoid churn and potential ESLint errors by code that is "owned" by Glean.

Reply:
One (small) harm is that people forget to regenerate it after changing/adding metrics and that then leads to additional steps during development or on a PR. You also need to ensure its generated from a compatible version of glean_parser.
Changes to autogenerated files are also not easy to review (as its all auto-generated and so might change with newer versions easily)
If you got that covered with appropriate tooling or docs that's fine, we're not gonna impose restrictions.

They should however be excluded from linting or auto-formatting.